### ISSUE TITLE
[Request] Microsoft Visual C++ Build Tools

### ISSUE THREAD

**Issue by fritzmg** [lukesampson/scoop-extras#1861]
Some applications like CodeIntel that is installed via `pip` (Python) require the _Microsoft Visual C++ Build Tools_. It would be handy if these could be installed via scoop as well.

**Comment by Ash258** [comment]
Are these build tools still installed using visual studio installer?

**Comment by fritzmg** [comment]
Yeah, I just noticed that they are installed via the Visual Studio Installer - so I am not sure if they can be downloaded/installed in a stand alone way …

**Comment by Ash258** [comment]
I was testing it few months back, but without success. Maybe someone will find way.

**Comment by fritzmg** [comment]
I just found [this](http://go.microsoft.com/fwlink/?LinkId=691126&fixForIE=.exe.) via https://stackoverflow.com/a/44953739/374996

**Comment by rasa** [comment]
See https://github.com/jberezanski/ChocolateyPackages/blob/master/visualstudio2019buildtools/tools/ChocolateyInstall.ps1 and 
https://github.com/jberezanski/ChocolateyPackages/blob/master/chocolatey-visualstudio.extension/extensions/Install-VisualStudio.ps1

**Comment by Adriankhl** [comment]
What about this https://github.com/felixrieseberg/windows-build-tools?

**Comment by arrebole** [comment]
I want `windows sdk` `msvc`

**Comment by karan-sorout** [comment]
Is this now added into extras? couldn't find it still, please help me with its name just in case I'm missing it.

**Comment by andrewrothstein** [comment]
scoop install rust-msvc

**Comment by milnak** [comment]
You can install Visual Studio Build Tools using:

```PowerShell
Invoke-WebRequest -Uri 'https://aka.ms/vs/17/release/vs_BuildTools.exe' -OutFile "$env:TEMP\vs_BuildTools.exe"

& "$env:TEMP\vs_BuildTools.exe" --passive --wait --add Microsoft.VisualStudio.Workload.VCTools --includeRecommended --remove Microsoft.VisualStudio.Component.VC.CMake.Project	
```

Don't include `--remove Microsoft.VisualStudio.Component.VC.CMake.Project` if you want CMake to be included.  Personally, I use `scoop install cmake` as that has a newer version.

**Comment by pcgeek86** [comment]
There's a conflict with the `busybox` package for `link.exe`, so I had to uninstall it. `scoop uninstall busybox`

**Comment by Cornelius-Figgle** [comment]
was there a reason why this isn't packaged with Scoop? Can it not be done with the Visual Studio installer?

**Comment by pidgeon777** [comment]
Hello, I wonder if there is any update, it would be cool to add the `Build Tools` to `scoop`.

**Comment by danlikestocode** [comment]
this would be great!

**Comment by niheaven** [comment]
Ref: #13027

**Comment by Eboubaker** [comment]
choco install python visualstudio2022-workload-vctools -y

**Comment by pidgeon777** [comment]
> choco install python visualstudio2022-workload-vctools -y

But this is for chocolatey, not scoop, right?

**Comment by Eboubaker** [comment]
> > choco install python visualstudio2022-workload-vctools -y
> 
> But this is for chocolatey, not scoop, right?

I want giving a reference that it can be done with a script. because someone said "Yeah, I just noticed that they are installed via the Visual Studio Installer - so I am not sure if they can even be downloaded/installed in a stand alone way …"
I updatedd my comment to quote that
